packages/cli/src/InternalHooks.ts

```diff
@@ -1,10 +1,10 @@
 import { Service } from 'typedi';
-import type { User } from '@db/entities/User';
 import { Telemetry } from '@/telemetry';
 import { MessageEventBus } from './eventbus/MessageEventBus/MessageEventBus';
 
 /**
- * @deprecated Do not add to this class. To add log streaming or telemetry events, use
+ * @deprecated Do not add to this class. It will be removed once we remove
+ * further dep cycles. To add log streaming or telemetry events, use
```

Comment:
We cannot yet fully remove this because of [this issue](https://n8nio.slack.com/archives/C069HS026UF/p1723195951853579), but effectively this is no longer used.